superset/utils/import_datasource.py

```diff
@@ -30,7 +29,7 @@ def import_datasource(
      superset instances. Audit metadata isn't copies over.
     """
     make_transient(i_datasource)
-    logging.info("Started import of the datasource: {}".format(i_datasource.to_json()))
+    logging.info("Started import of the datasource: %s", i_datasource.to_json())
```

Comment:
Can we use f strings instead?

Reply:
not according to pylint.

Reply:
fstrings and `.format` are not recommended for logging statements because they make you pay the cost of formatting the string regardless of logging level. Not that it's expensive, but it's best practice to use the format above for that reason